bin/wasabi.sh

```diff
@@ -347,26 +353,21 @@ for command in ${@:$OPTIND}; do
     bootstrap) bootstrap;;
     build) build true;;
     clean) clean;;
-    start) command="start:cassandra,mysql,wasabi";&
-    start:*) commands=$(echo ${command} | cut -d ':' -f 2)
-      (IFS=','; for command in ${commands}; do start ${command}; done);;
+    start) exec_commands start "cassandra,mysql,wasabi";;
+    start:*) exec_commands start ${command};;
     test:*) commands=$(echo ${command} | cut -d ':' -f 2)
       (IFS=','; for command in ${commands}; do mvn "-Dtest=com.intuit.wasabi.${command/-/}.**" test -pl modules/${command} --also-make -DfailIfNoTests=false -q ; done);;
```

Comment:
Maybe we can move this command to its own function as well, to streamline the `switch` completely?